README.md

```diff
@@ -1 +1,85 @@
-# budget_app
\ No newline at end of file
+# Budget app
+
+> Budget app is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+
+## Built With
+
+- RUBY
+- RUBY ON RAILS
+- Devise
+
+## Gems Used
+
+- Rubocop
+- RAILS (v7)
+
+## How to Setup
+
+> You can simply clone or download [this repository](https://github.com/mahbubul14/budget_app.git), and use your favorite browser or code editor to run this program.
+
+- To open this project using vs code ( for this example) or your favorite code editor, you can follow the guide below:
+  > in your cmd or command line navigate to where this project is located, then:
+
+```bash
+cd budget_app
+```
+
+> thereafter run
+
+```bash
+code .
+```
+
+## How to Run the App through terminal
+
+- To run the application through trminal, make sure ruby and rails is installed in your computer then follow the guide below:
+  > in your cmd or command line navigate to where this project is located, then;
+
+```cmd
+cd budget_app
+```
+
+## Install Gems
+
+
+```bash
+bundle install
+```
+
+> thereafter run the below command to start the server on localhost
+
+```bash
+rails s 
+```
+## Set up database
+> to create databases on your local machine follow the steps below :
+1. First make sure that postgresql installed on your machine
+2. make sure that postgresql server is runninng 
+3. run the following command to create database on your machine
+  ```bash
+  rake db:create
+  ```
+  ### Update schema file and database
+  - To run the migration files to update schema and databse simply run the following command :
+  ```bash
+  rails db:migrate
+  ```
+
+## Author
+
+👤 **Mahbub Alam**
+
+- GitHub: [@mahbubul14](https://github.com/mahbubul14)
+- Twitter: [@twitterhandle](https://twitter.com/mahbubul_14)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mahbubul14/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] The License section of the README from [this template](https://github.com/microverseinc/readme-template) is missing. Please let's include this. It will render the README file professional